src/browser_authentication_app_skeleton/src/actions/log_ins/delete.cr

```diff
@@ -0,0 +1,7 @@
+class LogIns::Delete < BrowserAction
+  delete "/sign_out" do
+    sign_out
```

Comment:
Same here. It seems a little out of place to have `Logins::Delete` call `sign_out`.